src/redis-cli.c

```diff
@@ -6541,9 +6541,9 @@ static int clusterManagerCommandImport(int argc, char **argv) {
     }
 
     if (config.cluster_manager_command.flags & CLUSTER_MANAGER_CMD_FLAG_COPY)
-        strcat(cmdfmt, " %s");
```

Comment:
WOW... what a strange approach...
conditionally concatenating an `%s` into the format string. and later unconditionally passing the "COPY" string argument to the formatter.

(not to mention the memory corruption of writing into unknown buffer size)